packages/aws-cdk-lib/core/lib/removal-policy.ts

```diff
@@ -48,6 +48,21 @@ export enum RemovalPolicy {
    * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html#aws-attribute-deletionpolicy-options
    */
   SNAPSHOT = 'snapshot',
+
+  /**
+   * Resource will be retained when they are requested to be deleted
+   * or need to be replaced due to an update request.
+   * Resource are not retained, if the creation is rolled back.
```

Comment:
Would this be too verbose? I'm on the fence. I think it's good to clarify that the resource can still be deleted if a delete request comes from something besides CFN. 

```suggestion
   * Resource will be retained when they are requested to be deleted during a stack delete request
   * or need to be replaced due to a stack update request.
   * Resource are not retained, if the creation is rolled back.
```